### PR TITLE
Revert edge glow effect and restore damped overscroll

### PR DESCRIPTION
**Pull request by suphon-t**
App drawer overscroll is already backported so this should be too.

### CODE REVIEW

File: app/src/main/java/ch/deletescape/lawnchair/PagedView.java [hunk 1]
```diff
@@ -31,6 +31,7 @@
 import android.os.Bundle;
 import android.util.AttributeSet;
 import android.util.DisplayMetrics;
+import android.util.Log;
```
[the remark this answers is not in the document]
Comment (divadsn) [reply app/src/main/java/ch/deletescape/lawnchair/PagedView.java:34]: Fuck

**Comment by divadsn** [comment]
@paphonb a toggle in the Settings would be nice to have, otherwise works fine.

File: app/src/main/java/ch/deletescape/lawnchair/PagedView.java [hunk 2]
```diff
@@ -1803,6 +1804,10 @@ protected void onScrollInteractionBegin() {
     }
 
     protected void onScrollInteractionEnd() {
+        if (mShowLawnfeedPopup) {
+            mShowLawnfeedPopup = false;
+            Utilities.showLawnfeedPopup(getContext());
```
Comment (divadsn) [inline app/src/main/java/ch/deletescape/lawnchair/PagedView.java:1809]: Is working, but how do we bring back the old fade effect?

File: app/src/main/java/ch/deletescape/lawnchair/PagedView.java [hunk 3]
```diff
@@ -1425,6 +1386,29 @@ public int getScrollForPage(int index) {
...
     }
 
+    // This curve determines how the effect of scrolling over the limits of the page dimishes
+    // as the user pulls further and further from the bounds
+    private float overScrollInfluenceCurve(float f) {
+        f -= 1.0f;
+        return f * f * f + 1.0f;
+    }
+
+    protected float acceleratedOverFactor(float amount) {
```
Comment (divadsn) [inline app/src/main/java/ch/deletescape/lawnchair/PagedView.java:1396]: We also might wanna check if the device is in battery save mode to disable spring animation.

**Review by divadsn — CHANGES_REQUESTED** [review]
Left some comments, otherwise nothing to complain.

File: app/src/main/java/ch/deletescape/lawnchair/PagedView.java (hunk 3 above)
Comment (suphon-t) [reply app/src/main/java/ch/deletescape/lawnchair/PagedView.java:1396]: Actually this is just damping the scroll distance, not as demanding as the spring animation.

**Pull request merged**